tests/sentry/receivers/test_featureadoption.py

```diff
@@ -64,151 +69,135 @@ def test_first_event(self):
-            project=self.project, platform="csharp", message="csharp error message"
+        event = self.store_event(
+            data={"platform": "csharp", "message": "csharp error message"},
+            project_id=self.project.id,
         )
-        event = self.create_event(group=group, data={"platform": "csharp"})
         event_processed.send(project=self.project, event=event, sender=type(self.project))
 
         csharp = FeatureAdoption.objects.get_by_slug(organization=self.organization, slug="csharp")
         assert csharp.complete
 
     def test_perl(self):
-        group = self.create_group(
-            project=self.project, platform="perl", message="perl error message"
+        event = self.store_event(
+            data={"platform": "perl", "message": "perl error message"}, project_id=self.project.id
         )
-        event = self.create_event(group=group, data={"platform": "perl"})
         event_processed.send(project=self.project, event=event, sender=type(self.project))
 
         perl = FeatureAdoption.objects.get_by_slug(organization=self.organization, slug="perl")
         assert perl.complete
 
     def test_elixir(self):
-        group = self.create_group(
-            project=self.project, platform="elixir", message="elixir error message"
+        event = self.store_event(
+            data={"platform": "elixir", "message": "elixir error message"},
+            project_id=self.project.id,
         )
-        event = self.create_event(group=group, data={"platform": "elixir"})
         event_processed.send(project=self.project, event=event, sender=type(self.project))
 
         elixir = FeatureAdoption.objects.get_by_slug(organization=self.organization, slug="elixir")
         assert elixir.complete
 
     def test_cfml(self):
-        group = self.create_group(
-            project=self.project, platform="cfml", message="cfml error message"
+        event = self.store_event(
+            data={"platform": "cfml", "message": "cfml error message"}, project_id=self.project.id
         )
-        event = self.create_event(group=group, data={"platform": "cfml"})
         event_processed.send(project=self.project, event=event, sender=type(self.project))
 
         cfml = FeatureAdoption.objects.get_by_slug(organization=self.organization, slug="cfml")
         assert cfml.complete
 
     def test_groovy(self):
-        group = self.create_group(
-            project=self.project, platform="groovy", message="groovy error message"
+        event = self.store_event(
+            data={"platform": "groovy", "message": "groovy error message"},
+            project_id=self.project.id,
         )
-        event = self.create_event(group=group, data={"platform": "groovy"})
         event_processed.send(project=self.project, event=event, sender=type(self.project))
 
         groovy = FeatureAdoption.objects.get_by_slug(organization=self.organization, slug="groovy")
         assert groovy.complete
 
-    def test_csp(self):
```

Comment:
Deleted this test because this functionality doesn't work at all, it was incorrectly passing before.

Reply:
This value is never present in production